src/main/java/com/management/HumanResources/model/Employee.java

```diff
@@ -9,14 +9,14 @@
 @AllArgsConstructor
 public class Employee {
 
-    private int id;
+    private long id;
     private String firstName;
     private String lastName;
     private String email;
-    private int salary;
+    private String salary;
```

Comment:
Why did you use `String` for the salary here?
I remember you saying something about it today but I did not fully understand.

Reply:
Yes so I eventually plan on making these back to int. Right now our database has it as a String with the $ sign attached. Good point tho, I can update the database now so it won't be a worry in the future.